Tests/Twig/Extension/SonataAdminExtensionTest.php

```diff
@@ -1217,23 +1217,27 @@ class="x-editable"
             ),
 
             array(
-                '<td class="sonata-ba-list-field sonata-ba-list-field-text" objectId="12345">
-                <div class="sonata-readmore" data-readmore-height="40" data-readmore-more="Read more" '.
-                'data-readmore-less="Close">A very very long string.</div>
-                </td>',
+                <<<'EOT'
+<td class="sonata-ba-list-field sonata-ba-list-field-text" objectId="12345">
+<div class="sonata-readmore" data-readmore-height="40" data-readmore-more="Read more" data-readmore-less="Close">a</div>
```

Comment:
I'm currently fighting against the `removeExtraWhitespace` method that i first didn't noticed. Actually, i have some spaces that are added or removed... I'm on it.